lib/talk_api_client.rb

```diff
@@ -78,14 +78,14 @@ def create_connection(adapter)
   end
 
   def initial_request
-    request('get', '').body.each do |_, resource|
+    request('get', '', accept_header: "application/json").body.each do |_, resource|
```

Comment:
This won't need to be different after zooniverse/Talk-Api@c46b359160e435aa5 is deployed